sdk/Lusid.Sdk.Tests/tutorials/Instruments/EquityExamples.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class EquityExamples: DemoInstrumentBase
+    {
+        [LusidFeature("F5-20")]
+        [Test]
+        public void EquityCreationAndUpsertionExample()
+        {
+            // CREATE an Equity (that can then be upserted into LUSID)
+            var equity = (Equity) InstrumentExamples.CreateExampleEquity();
+            
+            // ASSERT that it was created
+            Assert.That(equity, Is.Not.Null);
+            
+            // CAN NOW UPSERT TO LUSID
+            var uniqueId = equity.InstrumentType + Guid.NewGuid().ToString(); 
+            var instrumentsIds = new List<(LusidInstrument, string)>{(equity, uniqueId)};
+            var definitions = TestDataUtilities.BuildInstrumentUpsertRequest(instrumentsIds);
```

Comment:
Are you overwriting the LUID you create in `InstrumentExamples.CreateExampleEquity`?

Its not very clear from a user perspective

Reply:
what was the resolution?

Reply:
In the newest commit, I changed it to some ISIN
```
new EquityAllOfIdentifiers(isin: "US-000402625-0")
```

ok?